test/api_client/api_resource.js

```diff
@@ -89,8 +88,10 @@ describe('API resource module', () => {
 	describe('#request', () => {
 		let popsicleStub;
 		let handleRetryStub;
+		let defaultError;
 
 		beforeEach(() => {
+			defaultError = new Error('network error');
```

Comment:
I know this is in the tests, a little more verbose error still makes sense to me.
Something like`could not connect to a node` error